PokeAlarm/Utilities/QuestUtils.py

```diff
@@ -27,3 +27,56 @@ def get_reward_type(reward_type):
     except ValueError:
         raise ValueError("Unable to interpret `{}` as a valid "
                          " quest reward type or id.".format(reward_type))
+
+
+def reward_string(quest, locale):
+    if quest.reward_type_id == 7:
```

Comment:
Can you add some comments or clarification to this function? Not quite sure what the significance of these ids are, or why they mean different things.